### DISCUSSION TITLE
Regarding Feb 2024 cohort GCE application results

### DISCUSSION THREAD

**Discussion by VirtualHorror** [community/community#113330] [GitHub Education] [answered]
### Hi everyone,

I had applied for the GCE program about a month ago and received an email regarding the results of the application today morning.
Unfortunately, my application was not successful this time.

I was a bit surprised about this outcome since I had truly put in a lot of effort into this application - reflecting on feedback, refining essays and genuinely researching the challenges faced by the student community in my university & planning how to address them leveraging the resources provided by this program.
Additionally, in the previous cohort it was mentioned that my application was outstanding but, due to limited number of spots available in the cohort it was not possible to move forward with my application, which was fair.

While I understand that this cohort may have been extremely competitive with an overwhelming number of applications, I genuinely believe that I should have been provided with this opportunity since being a student from one of the top 5 universities in Silicon Valley/TechHub of India, I was passionate to lead and guide a student community of over 5000 students within the university and many more in the broader student community.
What further surprised me was that when I searched on X (formerly Twitter) regarding the results of the current cohort, all of the tweets & **specifically** of Indian students mentioned that their applications were not successful; wondering if anyone was even selected. They have all received the exact same feedback of "Have a deeper understanding of the struggles your community faces".
I'm not sure if this is because of an oversight or GitHub has entirely stopped supporting new GCE applications from India; which as per Satya Nadella is said to become the largest developer community on GitHub by 2027.

I understand that this is not the only program to support student communities and will certainly still be continuing to lead student communities helping enrich and grow the lives of others around me. 
But it's still disappointing because I truly believed that the resources provided by this program were valuable.
I fail to understand that if such genuinely interested applicants are not selected then who are?
If there is any possibility, I want to request for my application to be re-reviewed.
cc: @juanpflores @lasr21 @mvkaran @samus-aran @queenofcorgis

Had already worked on this and planned so much more, guess all this has to go now :(
![image](https://github.com/community/community/assets/35888729/5e2e52ad-ed4f-4ced-96fc-97e0d57700d7)
<br>
<br>
<br>
<br>
Attachments:
![image](https://github.com/community/community/assets/35888729/d951ad5e-ccb3-46a4-ae63-3ae839c6c853)
![image](https://github.com/community/community/assets/35888729/c683c7df-466a-452d-b815-09aca49d5ba1)
![image](https://github.com/community/community/assets/35888729/0f3deb8e-07f8-467d-a6a4-41f380d58dba)
![image](https://github.com/community/community/assets/35888729/55bb6c2d-df03-4dff-96ad-cecec286caeb)

**Comment by priyankarpal** [comment]
Although I am unsure about the reason for the rejection email, it would be helpful to receive a more detailed explanation.


![Screenshot_2024-03-20_20-16-00](https://github.com/community/community/assets/88102392/24c7fa29-27c1-4b9d-afc8-eb0507fd9464)

**Reply by VirtualHorror** [reply]
Don't know if they'll be able to provide more detailed feedback since they receive quite a bit of applications but what I know is that everyone or at least everyone I know and on X (formerly Twitter) especially from India have received the exact same generalized rejection reason which makes me question if this has been an oversight/some sort of technical glitch or is a legitimate rejection.

**Reply by priyankarpal** [reply]
yes, same doubt

**Comment by samus-aran** [comment, marked as answer]
Hi @VirtualHorror & all others in this thread 👋 

First and foremost, we want to acknowledge the exceptional quality of applicants who applied to the Campus Experts cohort program from 93 countries across the world. The talent and dedication demonstrated by all applicants were truly remarkable, making the selection process incredibly competitive. ⭐

We understand that receiving a rejection email can be disheartening, especially after investing your time and effort into the application. Please know that your feelings of disappointment are completely valid, and we empathize with you and everyone who was not selected.

However, we want to emphasize that this does not define your worth or potential. The reality is that despite how popular the program is, the number of spots available is extremely limited - we had thousands of applicants. As a result, many qualified applicants were not selected to progress further.  A large proportion of our successful candidates came from India.

We truly value your interest and participation in the application process, and we hope that you will continue to engage with us and explore other avenues for personal and professional development on GitHub.

I've included a link to another reply that we share some of the things the team look for in the applicants for the next time:
- https://github.com/orgs/community/discussions/113331#discussioncomment-8907172
